openhands/server/session/manager.py

```diff
@@ -214,10 +214,14 @@ async def join_conversation(self, sid: str, connection_id: str, settings: Settin
         logger.info(f'join_conversation:{sid}:{connection_id}')
         await self.sio.enter_room(connection_id, ROOM_KEY.format(sid=sid))
         self.local_connection_id_to_session_id[connection_id] = sid
-        event_stream = await self._get_event_stream(sid)
-        if not event_stream:
+        if True:
```

Comment:
is this condition intentional?

Reply:
Yes, I wanted to check with you if this is ok for other sandbox modes.

I confirm that it works, but I'm not sure if it's the best approach.

Reply:
Removed `sandbox.persist_sandbox_for_each_conversation` https://github.com/SmartManoj/Kevin/commit/8f3d8dab6354c2a7596f517677ba4410295361a4